simhash/__init__.py

```diff
@@ -105,20 +107,12 @@ def build_by_features(self, features):
                 w = f[1]
             for i in range(self.f):
                 v[i] += w if h & masks[i] else -w
-        ans = 0
-        for i in range(self.f):
-            if v[i] > 0:
-                ans |= masks[i]
-        self.value = ans
+        binary_str = ''.join(['0' if i <= 0 else '1' for i in v])
```

Comment:
Hi KK, thanks again for replying.

I see this is another improvement you are trying to make.
Could you move this change into a separate pull request?
I just want to make this change minimal so that it's easy to test and I'll have much higher confidence to merge if so.
Thanks.

Reply:
Okay, I've revert the value calculation.

Reply:
Thanks, bebound. Can you update the unit tests as well? Because right now the build is failing. I'm not sure why the build result is not showing here, but I'll check later. (By right, each commit will trigger a CI build and the result will show here)

Reply:
Sorry, I forget to rollback the value in test. It should work now.

Reply:
Travis said I forget to add `gmpy2` as dependency, I'll fix that.

Reply:
Travis has been fixed now.